### PR TITLE
Tidy up tutorials

### PR DESCRIPTION
**Pull request by deepakunni3**
This PR is a first pass at tidying up the tutorials after going through most of them.

This PR also fixes a bug in `SqlImplementation.__post_init__()` method, without which the examples in the tutorial (`README.md` and `introduction.md`) wouldn't work.

Questions:
- I see we are using Sphinx for documentation. Are we committed to Sphinx/reST?
- Can we look into MkDocs, since it has typically been used in most of our Python projects?

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
Awesome @deepakunni3!

**Comment by deepakunni3** [comment]
Thanks @matentzn ! 

@matentzn @cmungall I can't seem to assign any reviewers to this PR. 
Feel free to review or delegate to other devs on this repo :)

**Review by cmungall — CHANGES_REQUESTED** [review]

**Comment by cmungall** [comment]
Thanks!

my group best practice http://www.berkeleybop.org/best_practice is Mkdocs for ontology and schema centric repos and sphinx for code centric. You’ll see we make heavy use of autodoc here!

File: src/oaklib/implementations/sqldb/sql_implementation.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def __post_init__(self):
                 # they should be running through ODK docker
                 locator = locator.replace('.owl', '.db').replace('sqlite:', '')
                 semsql_builder.make(locator)
-                locator = f'sqlite:///{locator}'
+            locator = f'sqlite:///{locator}'
```
[the remark this answers is not in the document]
Comment (deepakunni3) [reply src/oaklib/implementations/sqldb/sql_implementation.py]: Yes, good point. I removed the code change from this PR 👍

**Comment by deepakunni3** [comment]
Thanks @cmungall ! That makes sense.

**Pull request merged**